app/controllers/application_controller.rb

```diff
@@ -1,23 +1,25 @@
 class ApplicationController < ActionController::Base
```

Comment:
we really need to find a way of preventing all these reformats, it's making it hard to review the PR. same goes for `Gemfile`